src/qibolab/qubits.py

```diff
@@ -38,14 +38,25 @@ class Qubit:
     name: QubitId
 
     bare_resonator_frequency: int = 0
-    readout_frequency: int = 0  # this is the dressed frequency
+    readout_frequency: int = 0
+    """ Readout dressed frequency"""
     drive_frequency: int = 0
     anharmonicity: int = 0
-    Ec: int = 0
-    Ej: int = 0
-    g: int = 0
+    sweetspot: float = 0.0
+    flux_to_bias: float = 0.0
+    asymmetry: float = 0.0
+    brf: float = 0.0
```

Comment:
@DavidSarlle what is the final decision for this? Here we are still using `brf` but in qiboteam/qibolab_platforms_qrc#54 you changed to `bare_resonator_frequency_sweetspot`.

Reply:
Depending on this I will also update https://github.com/qiboteam/qibocal/pull/538
Personally I think that we should use `bare_resonator_frequency_sweetspot`.
Also `ssf_brf` could be renamed to something like `ratio_sweetspot_qubit_freq_bare_resonator_freq` which is pretty long but at least we know what it is.

Reply:
Sorry @stavros11 and @andrea-pasquale I forgot to push the latest changes where I modified the brf name to bare_resonator_frequency_sweetspot.

@andrea-pasquale regarding the ssf_brf, we can modifiy it as you said, but is a very large name for a variable... I have also included the docstrings with the explanation, so I think is better if we keep it as it is...

Reply:
I agree, I will not argue for the name of a single variable.
Let's keep `ssf_brf` and merge this PR.

Reply:
Great, if everyone agrees, please merge this when the CI passes.